docs/docs-content/integrations/hello-universe.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: "Hello Universe"
+title: "Hello Universe"
+description: "Learn about the Hello Universe pack and how you can use it within your Kubernetes clusters."
+hide_table_of_contents: true
+type: "integration"
+category: ["app services", "amd64"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: "https://raw.githubusercontent.com/spectrocloud/pack-central/main/packs/hello-universe-1.1.1/logo.png"
```

Comment:
The Hello Universe pack is stored in the Palette OCI registry, which is hosted on Amazon ECR. The API payload of this pack does not have the `logo` line as we do with the packs of the Public Registry. Therefore, the GitHub URL was used to get the pack's logo.